aerogear/mobile-developer-console
# Delete mobileclient functionality

## Motivation
https://issues.jboss.org/browse/AEROGEAR-7961

## What
Allow user to delete Mobile clients

## Why
Because it's annoying to do it always via OpenShift console.

## How
Similarly as in https://github.com/aerogear/mobile-developer-console/pull/63

## Verification Steps
💁‍♂️  Feel free to use [my project on community cluster](https://community.skunkhenry.com/console/project/psturc-test/overview) (ping me for user/pass if you don't have it)
1. Clone this branch and follow instructions [from here](https://github.com/aerogear/mobile-client-service/blob/master/README.md#local-development)
1. In MDC, go to main page (`/overview`), create new mobile client and then try to delete it.
1. Mobile client should disappear, no error message should be present in console.

## Checklist:
